test/fork/wstETHOracle.ts

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-restricted-syntax */
+import { smock } from "@defi-wonderland/smock";
+import chai from "chai";
+import { Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers, upgrades } from "hardhat";
+
+import { contracts as ethereum } from "../../deployments/ethereum.json";
+import { contracts as sepolia } from "../../deployments/sepolia.json";
+import { ResilientOracle, WstETHOracle__factory } from "../../typechain-types";
+import { TokenConfigStruct } from "../../typechain-types/contracts/ResilientOracle";
+import { WstETHOracle } from "../../typechain-types/contracts/oracles/WstETHOracle";
+import { addr0000 } from "../utils/data";
+import WSTETH_ABI from "./abi/wstETH.json";
+import { forking, initMainnetUser } from "./utils";
+
+const { expect } = chai;
+chai.use(smock.matchers);
+
+const FORK: boolean = process.env.FORK === "true";
+const FORKED_NETWORK: string = process.env.FORKED_NETWORK || "";
+const WETH_USD_PRICE_DENOMINATOR = parseUnits("1", 18);
+
+const oracleAddress = {
+  ethereum: ethereum.ResilientOracle.address,
+  sepolia: sepolia.ResilientOracle.address,
+};
+
+type BlockConfig = {
+  [key: string]: number;
+};
+
+type AddressConfig = {
+  [key: string]: string;
+};
+
+const stETHAddress: AddressConfig = {
+  ethereum: "0xae7ab96520de3a18e5e111b5eaab095312d7fe84",
+  sepolia: "0xF5465B70Af90AEb26Aa13b1000a8CbEA53a5f4cf",
+};
+
+const wstETHAddress: AddressConfig = {
+  ethereum: "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0",
+  sepolia: "0x9b87ea90fdb55e1a0f17fbeddcf7eb0ac4d50493",
+};
+
+const WETHAddress: AddressConfig = {
+  ethereum: "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+  sepolia: "0x700868CAbb60e90d77B6588ce072d9859ec8E281",
+};
+
+const contractAdmin: AddressConfig = {
+  ethereum: "0x285960C5B22fD66A736C7136967A3eB15e93CC67",
+  sepolia: "0x94fa6078b6b8a26f0b6edffbe6501b22a10470fb",
+};
```

Comment:
Would it make sense to get these addresses from `helpers/deploymentConfig.ts` (except the `WETHAddress`)?